src/wallet/walletdb.cpp

```diff
@@ -597,6 +600,7 @@ DBErrors CWalletDB::LoadWallet(CWallet* pwallet)
 
     LogPrintf("Keys: %u plaintext, %u encrypted, %u w/ metadata, %u total\n",
            wss.nKeys, wss.nCKeys, wss.nKeyMeta, wss.nKeys + wss.nCKeys);
+    LogPrintf("Unknown wallet records: %u\n", wss.m_unknown_records);
```

Comment:
To reduce the amount of logging I suggest printing this only if `wss.m_unknown_records != 0`.

Alternatively add `wss.m_unknown_records` printing to the already existing log entry (`Keys: …`) above.

Reply:
> To reduce the amount of logging I suggest printing this only if wss.m_unknown_records != 0.

Or print it on the same line as the `Keys:`. In that case it doesn't matter to log it every time.

Reply:
@laanwj Agreed! That was suggestion 2 :-) My vote is on suggestion 2.

Reply:
👍 @practicalswift

Reply:
I split it out because the print line is "Keys:", but I suppose this could have meant "Key/Value" versus "crypto keys"?

Reply:
It's counting only the crypto keys as "total", not metadata which I think led me to believe this.

If I add it to the above line, I suppose I'll ommit it from "total"?

Reply:
Indeed, I didn't mean to include it in keys but have it seperately e.g.
```
Keys: %u plaintext, %u encrypted, %u w/ metadata, %u total. Unknown wallet records: %u.
```